public/data/javascript.json

```diff
@@ -906,6 +1084,38 @@
         ],
         "tags": ["javascript", "localStorage", "storage", "utility"],
         "author": "dostonnabotov"
+      },
+      {
+        "title": "Check if Item Exists in localStorage",
+        "description": "Checks if a specific item exists in localStorage.",
+        "code": [
+          "const isItemInLocalStorage = (key) => {",
+          "  return localStorage.getItem(key) !== null;",
+          "};",
+          "",
+          "// Usage:",
+          "console.log(isItemInLocalStorage('user')); // Output: true or false"
+        ],
+        "tags": ["javascript", "localStorage", "storage", "utility"],
+        "author": "axorax"
+      },
+      {
+        "title": "Check bytes used",
```

Comment:
I still find this one weird, it doesn't return the quantity of bytes used, IMO just remove it, there is no way in JS of knowing the length in bytes of a type, and the localstorage length can already be accessed